server/ServerApp.cpp

```diff
@@ -1344,6 +1345,11 @@ void ServerApp::LoadGameInit(const std::vector<PlayerSaveGameData>& player_save_
             ErrorLogger() << "LoadGameInit got inconsistent empire ids between player save game data and result of PlayerEmpireID";
         }
 
+        // Revoke readiness only for online players so them could re-make orders of current turn.
```

Comment:
so they can redo order for the current turn.

Reply:
Fixed.